extensions/dacpac/src/test/wizard.test.ts

```diff
@@ -70,4 +106,49 @@ describe('Dacfx wizard', function (): void {
 		wizard.setDoneButton(Operation.export);
 		should.equal(wizard.selectedOperation, Operation.export);
 	});
+
+	it('Should start all pages - Set 1', async () => {
+		testContext = createContext();
+		wizard.setPages();
+		let result = false;
+
+		let selectOperationPage = new SelectOperationPage(wizard, wizard.pages.get(PageName.selectOperation).wizardPage, wizard.model, testContext.viewContext.view);
+		result = await selectOperationPage.start();
```

Comment:
can there be a helper function that starts a page and validates the result so result doesn't have to keep getting set to false?